earth_enterprise/docs/landing_page/manual/answer/3230746.html

```diff
@@ -130,7 +129,7 @@
   <h2><a id="ServePortable" name="ServePortable">Serve a globe or map from GEE Portable</a></h2>
   <p>Installing GEE Portable creates a <code>Google_Earth_Portable</code> directory on your machine. This directory contains a folder called <code>maps</code> (unless you renamed it in <code>portable.cfg</code>). Copy your globe or map to the <code>maps</code> folder. The GEE Portable interface lists all the globes and maps that you place in this folder. If you no longer want a globe or map to appear in the list, simply remove it from the <code>maps</code> folder.</p>
   <h2><a id="ServeServer" name="ServeServer">Serve a globe or map from the Google Earth Enterprise Server</a></h2>
-  <p>If you want to serve a globe or map to a large number of users, you can use a Google Earth Enterprise Server (GEE Server) on a Linux machine instead of buying a GEE Portable license for each user’s machine. A GEE Server license allows 10,000 users. In contrast, each GEE Portable license allows either 1 or 10 users. Another reason to use a GEE Server is that it's capable of storing very large globes or maps. GEE Servers also let you serve globes and maps on your own private network so that only authorized users can connect.</p>
+  <p>If you want to serve a globe or map to a large number of users, you can use a Google Earth Enterprise Server (GEE Server) on a Linux machine instead of a GEE Portable license on each user’s machine.</p>
```

Comment:
I think this sentence could be reworded:
1. Since there aren't really Portable licenses any more, we should reword that phrase.
1. The second half of the line is still useful.

The resulting line would be something like:

If you want to serve a globe or map to a large number of users, you can use a Google Earth Enterprise Server (GEE Server) on a Linux machine instead of a GEE Portable Server on a user’s machine.  GEE Server is capable of storing very large globes or maps, and it also lets you serve globes and maps on your own private network so that only authorized users can connect.